src/content/glossary/index.md

```diff
@@ -34,10 +34,20 @@ In [Solidity](#solidity), `assert(false)` compiles to `0xfe`, an invalid opcode,
 
 <DocLink to="/developers/docs/security/" title="Security" />
 
+### attestation {#attestation}
+
+A validator vote for a [Beacon Chain](#beacon-chain) or [shard](#shard) [block](#block). Validators must attest to blocks, signalling that they agree with the state proposed by the block.
```

Comment:
If I am correct, the spelling of "signalling" vs "signaling" is regional. We only have "en" without any differentiation for British vs American, but correct me if I'm wrong, words like "organization" vs "organisation" on the site tend to be the American spelling?
For consistency would suggest changing to `signaling`

Reply:
That's a good point, there's quite a mix of English (British vs. American) across the site. Probably makes sense to pick one to go with.

Reply:
Not sure what the best way to decide, e.g. if [Google Trends data](https://trends.google.com/trends/explore?q=organization,organisation) is an accurate signal of understanding/demand:
<img width="1163" alt="Image 2020-12-01 at 9 17 24 AM" src="https://user-images.githubusercontent.com/8097623/100752057-f4359880-339c-11eb-9d96-590e35a7648b.png">